source/features/link-to-compare-diff.tsx

```diff
@@ -0,0 +1,26 @@
+import './link-to-compare-diff.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {wrapAll} from '../helpers/dom-utils';
+
+function init(): void {
+	const changedFilesSummary = select('.Box .octicon-file-diff')!.closest('li')!;
+	wrapAll(
+		[...changedFilesSummary.children],
```

Comment:
Can `wrapAll` be adjusted to support this? It doesn't seem necessary
```suggestion
	wrapAll(
		changedFilesSummary.children,
```

Reply:
The problem is that the returned `HTMLCollection` is updated when a new child is inserted, so the wrapper will become part of it and try to append itself to itself:

```
❌ link-to-compare-diff 0.0.0 → DOMException: Element.append: The new child is an ancestor of the parent refined-github.js:8091:11
```

Reply:
ur-right 🥲